mkdocs.yml

```diff
@@ -11,17 +11,20 @@ pages:
 - Usecases: biz/usecases.md
 
 - Setup:
-  - Fabric Developer Setup: dev-setup/devenv.md
-  - Chaincode or Application Developer Setup: Setup/Chaincode-setup.md
-  - Java Chaincode Setup: Setup/JAVAchaincode.md
+  - Development Environment: dev-setup/devenv.md
```

Comment:
Why did you rename this? I named it specifically to denote that this is the development environment for a Fabric developer. It really is not and should not be the default for application and chaincode developers.